src/axom/inlet/Inlet.hpp

```diff
@@ -57,10 +59,11 @@ class Inlet : public SchemaCreator
    *****************************************************************************
    */
   Inlet(std::shared_ptr<Reader> reader,
-        axom::sidre::Group* sidreRootGroup) :
+        axom::sidre::Group* sidreRootGroup, bool docsEnabled = false) :
```

Comment:
@white238 -- can you please clarify something -- is `sidreRootGroup` the root group of the datastore (e.g. `DataStore::getRoot()` ), or is it the root group for `Inlet`?